layouts/partials/head.html

```diff
@@ -0,0 +1,45 @@
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>{{ .Title }}</title>
+
+  {{ $style := resources.Get "scss/index.scss" }} {{ $options := dict
+  "transpiler" "dartsass" }} {{ $style := $style | resources.ToCSS $options}}
```

Comment:
```suggestion
  {{ $style := resources.Get "scss/index.scss" }}
  {{ $options := dict "transpiler" "dartsass" }}
  {{ $style := $style | resources.ToCSS $options}}
```

Reply:
I'm actually gonna change this to use the `libsass` compiler.  It also looks like prettier did this formatting change (assuming this was a formatting suggestion)

Reply:
I'm gonna walk back the `libsass` change as it's been marked deprecated by the sass team :sweat: